lib/matplotlib/table.py

```diff
@@ -146,6 +147,57 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
 
 
+class CustomCell(Cell):
+    """
+    A subclass of Cell where the sides may be visibly toggled.
+
+    """
+
+    EDGES = 'BRTL'
+    EDGE_ALIASES = {'open':         '',
+                    'closed':       EDGES,  # default
+                    'horizontal':   'BT',
+                    'vertical':     'RL'
+                    }
+
+    def __init__(self, *args, **kwargs):
+
+        visible_edges = kwargs.pop('visible_edges')
+        Cell.__init__(self, *args, **kwargs)
+
+        if visible_edges is None:
+            self._visible_edges = self.EDGES
+        elif visible_edges in self.EDGE_ALIASES:
+            self._visible_edges = self.EDGE_ALIASES[visible_edges]
+        else:
+            for edge in visible_edges:
+                if edge not in self.EDGES:
+                    msg = 'Invalid edge param {0}, must only be one of {1} or \
+                            string of {2}.'.format(
+                                    visible_edges,
+                                    ", ".join(self.EDGE_ALIASES.keys()),
+                                    ", ".join(self.EDGES),
+                                    )
+                    raise ValueError(msg)
+            self._visible_edges = visible_edges
+
+    def get_path(self):
+        'Return a path where the edges specificed by _visible_edges are drawn'
+
+        codes = [Path.MOVETO] * 4
+        codes.append(Path.CLOSEPOLY)
```

Comment:
Hmm I see what you mean. We've actually hit a small bug with `CLOSEPOLY` in some extra testing we're doing and will report back with news soon/